ssl/ssl_lib.c

```diff
@@ -1685,6 +1740,9 @@ static int ssl_start_async_job(SSL *s, struct ssl_async_args *args,
         s->waitctx = ASYNC_WAIT_CTX_new();
         if (s->waitctx == NULL)
             return -1;
+        if (s->async_cb &&
```

Comment:
Expliclity compare this to NULL, i.e. `s->async_cb != NULL`. Put the `&&` at the beginning of the next line.

Reply:
done